commands/operator-sdk/cmd/build.go

```diff
@@ -172,7 +172,7 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 
 	log.Infof("Building Docker image %s", baseImageName)
 
-	dbArgs := []string{"build", ".", "-f", "build/Dockerfile", "-t", baseImageName}
+	dbArgs := []string{"build", "build/", "-t", baseImageName}
```

Comment:
I think with this change, the build command will work with both docker and podman. It seems like podman is a bit more particular about the order of the arguments.

```suggestion
	dbArgs := []string{"build", "-f", "build/Dockerfile", "-t", baseImageName, "."}
```